spec/features/user_views_category_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+feature 'user views article', %{
+  As a user, I want to view a list of everything within a category, so that I can
+  have an overview of the content of that category.
+  Acceptance Criteria:
+  * [X] - I can see a list of all articles within a category.
+  * [ ] - I can see a list of all galleries within a category.
+  * [X] - I can see a list of all subcategories within a category.
+  * [X] - I can see a list of all articles within each subcategory listed.
+  * [ ] - I can see a list of all galleries within each subcategory listed.
+  } do
+
+  let(:category) do
+    FactoryGirl.create(:category)
+  end
+
+  let(:subcategory) do
+    FactoryGirl.create(:category, name: "Subcat", parent: category)
+  end
+
+  let(:article) do
+    FactoryGirl.create(:article, category: subcategory)
+  end
+
+  scenario 'user views category' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.